### DISCUSSION TITLE
Boas-vindas às Discussões do Brazilian Utils Python!

### DISCUSSION THREAD

**Discussion by camilamaia** [brazilian-utils/brutils-python#30] [Anúncios]
## 👋 Olar!

Estamos usando as Discussões como um lugar para conectar com outros membros da nossa comunidade. Esperamos que você:

* Faça perguntas sobre as quais esteja curioso.
* Compartilhe ideias.
* Interaja com outros membros da comunidade.
* Dê boas-vindas a outras pessoas e seja mente aberta. Lembre-se que esta é uma comunidade que construímos juntos 💪.

Para começar, comente abaixo com uma introdução sobre você e nos conte o que faz com esta comunidade.

**Comment by leowebc** [comment]
Gostaria de informar que a library está retornando CEP, inválidos! Gostaria muito de usar a Librady

**Reply by camilamaia** [reply]
Oii, obrigada pelo aviso. Você poderia abrir uma issue [aqui](https://github.com/brazilian-utils/brutils-python/issues) comentando mais do problema?

Quem sabe dar alguns exemplos para que possamos verificar melhor.

Valeu!
